projects/dot-custom-elements/src/components/dot-dropdown/dot-dropdown.tsx

```diff
@@ -0,0 +1,53 @@
+import { Component, Prop, State, Event, EventEmitter } from '@stencil/core';
+import { generateId } from '../../utils/utils';
+import Fragment from 'stencil-fragment';
+
+@Component({
+    tag: 'dot-dropdown',
+    styleUrl: 'dot-dropdown.scss'
+})
+export class DotDropdownComponent {
+    @Prop() label: string;
+    @Prop() hint: string;
+    @Prop() options: string;
+    @Prop() value: string;
+    @Event() onCallback: EventEmitter;
+
+    @State() _options: string[];
+    @State() _value: string;
+    _label: string;
+
+    componentWillLoad() {
+        this._options = this.options
```

Comment:
Whatever we use here to split the options and stuff, we might want to put it in a util file so we can reuse in other components.